frontend/src/components/Public.vue

```diff
@@ -1,166 +1,184 @@
+      existingRouteLayers: L.Polyline[];
+      initialized: boolean;
+      legend: L.Control;
+    };
   },
   mounted() {
-    const a  = new InfoService();
-    this.$store.dispatch('grabRoutes');
-    this.$store.dispatch('grabStops');
-    this.$store.dispatch('grabVehicles');
-    this.$store.dispatch('grabUpdates');
-    this.$store.dispatch('grabAdminMesssage');
+    const a = new InfoService();
+    this.$store.dispatch("grabRoutes");
+    this.$store.dispatch("grabStops");
+    this.$store.dispatch("grabVehicles");
+    this.$store.dispatch("grabUpdates");
+    this.$store.dispatch("grabAdminMesssage");
     setInterval(() => {
-      this.$store.dispatch('grabUpdates');
+      this.$store.dispatch("grabUpdates");
     }, 5000);
 
     this.$nextTick(() => {
       this.ready = true;
-      this.Map = L.map('mymap', {
+      this.Map = L.map("mymap", {
         zoomControl: false,
-        attributionControl: false,
+        attributionControl: false
       });
 
       this.Map.setView([42.728172, -73.678803], 15.3);
 
-      this.Map.addControl(L.control.attribution({
-          position: 'bottomright',
-          prefix: '',
-      }));
-      L.tileLayer('https://stamen-tiles.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}{r}.png', {
-        attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, ' +
-                      'under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. ' +
-                      'Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under ' +
-                      '<a href="http://www.openstreetmap.org/copyright">ODbL</a>.',
-        maxZoom: 17,
-        minZoom: 14,
-      }).addTo(this.Map);
+      this.Map.addControl(
+        L.control.attribution({
+          position: "bottomright",
+          prefix: ""
+        })
+      );
+      L.tileLayer(
+        "https://stamen-tiles.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}{r}.png",
+        {
+          attribution:
+            'Map tiles by <a href="http://stamen.com">Stamen Design</a>, ' +
+            'under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. ' +
+            'Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under ' +
+            '<a href="http://www.openstreetmap.org/copyright">ODbL</a>.',
+          maxZoom: 17,
+          minZoom: 14
+        }
+      ).addTo(this.Map);
 
       this.Map.invalidateSize();
       this.showUserLocation();
-
     });
     this.renderRoutes();
     this.$store.subscribe((mutation: any, state: any) => {
-      if (mutation.type === 'setRoutes') {
+      if (mutation.type === "setRoutes") {
         this.renderRoutes();
         this.updateLegend();
       }
-      if (mutation.type === 'setStops') {
+      if (mutation.type === "setStops") {
         this.renderStops();
       }
-      if (mutation.type === 'setVehicles') {
+      if (mutation.type === "setVehicles") {
         this.addVehicles();
       }
     });
+
+    new Fusion();
```

Comment:
I think this should create a fusion object and call a start() function, instead of just using the constructor. I would rather not have the constructor performing an ongoing operation and have it be explicitly called instead.